### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by graydon**
3 minor changes that came up while working on batch mode. The flycheck fix is quite optional but the stats dir changes are somewhat more important for measuring on linux and writing visually-useful catapult traces.

### CODE REVIEW

**Comment by graydon** [comment]
@swift-ci please smoke test

File: .dir-locals.el [hunk 1]
```diff
@@ -20,7 +20,8 @@
   (fill-column . 80)
   (c-file-style . "swift"))
  (c++-mode
-  (whitespace-style face lines indentation:space))
+  (whitespace-style face lines indentation:space)
+  (flycheck-clang-language-standard . "c++14"))
```
Comment (jrose-apple) [inline .dir-locals.el]: Will this mess up people who don't have flycheck installed?

**Review by jrose-apple — COMMENTED** [review]
I don't think I'm particularly qualified to review any of this, but nothing looks terribly out of place, and I guess it's good to get reviews.

File: .dir-locals.el (hunk 1 above)
Comment (graydon) [reply .dir-locals.el]: @jrose-apple not as far as I know (I have read manual attempting to tell, given up, and tried here with a no-local-state emacs and it seems harmless)

**Pull request merged**